tensorflow/core/kernels/data/experimental/snapshot_util.cc

```diff
@@ -905,7 +907,7 @@ Status DetermineOpState(const std::string& mode_string, bool file_exists,
   }
 
   if (metadata->creation_timestamp() >=
-      (static_cast<int64>(EnvTime::NowMicros()) -
+      static_cast<int64>(static_cast<int64>(EnvTime::NowMicros()) -
```

Comment:
There is already a cast? Maybe extract RHS to separate expression?

Reply:
updated.